openghg/standardise/_surface.py

```diff
@@ -0,0 +1,157 @@
+from pathlib import Path
```

Comment:
Does this need to be split out from the other standardise functions? And if so, should all the different data types be split out into their own module?

Reply:
I think it might be tidier instead of one big file, what do you think?

Reply:
I'm just thinking this may cause some difficult merge conflicts at the moment with `new_object_store` so if we could hold off on making that kind of change that may be better? I suppose it does depend on how the change has been implemented.